alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -51,6 +52,7 @@ class embedPlaykitJsAction extends sfAction
 	private $playerConfig = null;
 	private $uiConfUpdatedAt = null;
 	private $regenerate = false;
+	private $includeSourceMap = "false";
```

Comment:
Use ' for any string without vars inside

Reply:
Done.